.github/workflows/notification.yml

```diff
@@ -0,0 +1,105 @@
+name: Slack notifications for PRs and Issues
+
+on:
+  pull_request:
+    types: [opened, reopened]
+  issues:
+    types: [opened, reopened]
+
+env:
+  CHANNEL_WALLET_FEEDBACK: ${{ vars.CHANNEL_WALLET_FEEDBACK }}
+  CHANNEL_WALLET_SQUAD_BUILD: ${{ vars.CHANNEL_WALLET_SQUAD_BUILD }}
+  ON_CALL_WALLET: ${{ vars.ON_CALL_WALLET }}
+  ON_CALL_BUILD_SQUAD: ${{ vars.ON_CALL_BUILD_SQUAD }}
```

Comment:
![image](https://user-images.githubusercontent.com/966662/217946851-12d598a1-3d8b-4e75-8b70-5d25ab4ce8c7.png)

Reply:
currently they point to a test channel / user group